### ISSUE TITLE
Compendiums can be associated with multiple organisms

### ISSUE THREAD

**Issue by arielsvn** [AlexsLemonade/refinebio-frontend#776]
Once https://github.com/AlexsLemonade/refinebio/pull/1753 gets merged we'll include multiple organisms in some compendiums. Examples of the groups here https://github.com/AlexsLemonade/refinebio/issues/1736#issuecomment-541183861

@dvenprasad we might want to update the compendia page so people know that they might be getting multiple organisms in their compendia.

**Comment by dvenprasad** [comment]
![Screen Shot 2019-10-30 at 11 46 33 AM](https://user-images.githubusercontent.com/15315514/67876828-e9ac2e00-fb0e-11e9-8599-c198516db586.png)

Adding an info badge saying that compendia contains other organisms. **View list** links to the section in the docs added by https://github.com/AlexsLemonade/refinebio-docs/issues/116
I tried listing the organism list inline but some of them were a lot and it messed with the layout. 

I'm not super convinced about referring to the compendia by species name, when we say it aggregated by genus. I'm curious to see how users will respond to it. I'm okay with referring to it by species name and reiterating after testing, if needed. 

Copy: 
`Normalized Compendia are the collection of all the samples available on refine.bio, aggregated and normalized by genus.`

`Also contains small number of samples from other organisms from the same genus. View list` 

@jaclyn-taroni I just changed `normalized by species` to `normalized by genus` Is there additional information we need to add?  Also, any other comments on the copy? 



In **future iterations**, we should consider something like this, which opens up the layout and we can easily add information for each organism as needed. Just a thought. We should test how the current layout performs before we make this drastic change.

![IMG_20191030_120734 (1)](https://user-images.githubusercontent.com/15315514/67883195-0437d480-fb1a-11e9-942d-2bd32afc707f.jpg)
